recipes/quantumespresso/ci.yml

```diff
@@ -1,7 +1,18 @@
 include: '/ci/common.yml'
 
-build-a100:
-  extends: .stack-build-hohgant-a100
+#build-a100:
```

Comment:
Don't forget to uncomment this.

Will the GPU ReFrame tests be added in a separate PR?

Reply:
Yes, I want to test CPUs first and be ready for Eiger.